src/bundle/Resources/config/views.yml

```diff
@@ -5,3 +5,8 @@ system:
                 default:
                     template: "EzSystemsHybridPlatformUiBundle::locationview.html.twig"
                     match: true
+            versions:
```

Comment:
We should have a convention for tab views. I suggest locationview_XXX_tab. Example: `locationview_versions_tab`. Naming is debatable (ping @dpobel).